src/Microsoft.ML.LightGBM.StaticPipe/Microsoft.ML.LightGBM.StaticPipe.csproj

```diff
@@ -6,7 +6,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.ML.Data\Microsoft.ML.Data.csproj" />
-    <ProjectReference Include="..\Microsoft.ML.LightGBM\Microsoft.ML.LightGBM.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.LightGBM\Microsoft.ML.LightGbm.csproj" />
```

Comment:
You changed the casing of the folder, right? `\Microsoft.ML.LightGBM\` => `\Microsoft.ML.LightGbm\` or else the Linux build is gonna fail.